src/components/activity-history-list/activity-history-list.tsx

```diff
@@ -73,20 +75,31 @@ export const ActivityHistoryList = ({
                 </Thead>
                 <Tbody>
                     {activityHistory.map((activity, index) => {
-                        const dataChange = updatedData(activity);
-                        if (!dataChange) return null;
-
-                        return (
-                            <Tr
-                                key={index}
-                                className="govuk-table__row mtfh-activity-history"
-                            >
-                                <Td>{formattedDate(activity.createdAt)}</Td>
-                                <Td>{entityEdited(activity.targetType)}</Td>
-                                <Td>{dataChange}</Td>
-                                <Th>{activity.authorDetails.fullName}</Th>
-                            </Tr>
-                        );
+                        const { targetType } = activity;
```

Comment:
Think we agreed this wasn't a real type guard and that `type === 'person'` was sufficient.

Reply:
Yeah you are right. Removed guard and defaulting to
`targetType === 'person'` format